Formula/xsd.rb

```diff
@@ -4,6 +4,7 @@ class Xsd < Formula
   url "https://www.codesynthesis.com/download/xsd/4.0/xsd-4.0.0+dep.tar.bz2"
   version "4.0.0"
   sha256 "eca52a9c8f52cdbe2ae4e364e4a909503493a0d51ea388fc6c9734565a859817"
+  license "GPL-2.0-only"
```

Comment:
NOTE, there is some license exception deal in here, did not find anything related in the SDPX exception list

```
In addition, as a special exception, Code Synthesis Tools CC gives
permission to link this program with the Xerces-C++ library (or with
modified versions of Xerces-C++ that use the same license as Xerces-C++),
and distribute linked combinations including the two. You must obey
the GNU General Public License version 2 in all respects for all of
the code used other than Xerces-C++. If you modify this copy of the
program, you may extend this exception to your version of the program,
but you are not obligated to do so. If you do not wish to do so, delete
this exception statement from your version.

In addition, Code Synthesis Tools CC makes a special exception for
the Free/Libre and Open Source Software (FLOSS) which is described
in the accompanying FLOSSE file.
```